includes/utils/class-amp-dom-utils.php

```diff
@@ -499,4 +499,19 @@ public static function copy_attributes( $attributes, DOMElement $from, DOMElemen
 			}
 		}
 	}
+
+	/**
+	 * Get closest parent element for specified DOM node.
+	 *
+	 * @param DOMNode $node DOMNode.
+	 * @return DOMElement|null
+	 */
+	public static function get_parent_element( DOMNode $node ) {
```

Comment:
Is this needed? It's only being used to get the parent node of `DOMElement` instances, which can only have other `DOMElement` instances as parents, except in the case of the root element, right? (It won't ever be a `DocumentFragment` in our usage.) So you could then just check if `$node->parentNode` instance of `DOMElement` as opposed to checking if truthy.